File: day01/EvenOdd.java
import java.util.Scanner;

public class EvenOdd {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        while (true){
            System.out.print("Enter a whole number (no decimals): ");

            try{
                int user_input = scanner.nextInt();

                if (user_input % 2 == 0){
                    System.out.println("The number is even.");
                } else {
                    System.out.println("The number is odd.");
                }
                break;
              
            } catch (Exception e) {
                System.out.println("Please enter a whole number.");
                scanner.nextLine();
            }
        }
        scanner.close();
    }
}

File: day01/evenOdd.py
while True:
    user_input = input("Enter a whole number (no decimals): ")
    try:
        number = int(user_input)
        if number % 2 == 0:
            print("The number is even.")
        else:
            print("The number is odd.")
        break

    except ValueError:
        print("Please enter a whole number.")
